test/addons/dlopen-ping-pong/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../../common');
+
+if (common.isWindows) {
+  common.skip('dlopen global symbol loading is not supported on this os.');
+  return;
```

Comment:
The `return`s are not necessary. `skip` will take care of it.

Reply:
OK.